apps/foundation-demo/src/app/checkout/components/shipping-summary/shipping-summary.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="shipping-summary">
+  <div class="shipping-summary__title">shipping details</div>
+  <div class="shipping-summary__name">{{shippingInfo.firstname}} {{shippingInfo.lastname}}</div>
+  <div class="shipping-summary__street">{{shippingInfo.street}}</div>
+  <div class="shipping-summary__cityState">{{shippingInfo.city}}, {{shippingInfo.state}} {{shippingInfo.postcode}}</div>
+  <div class="shipping-summary__telephone">{{shippingInfo.telephone}}</div>
+  <a (click)="onEdit()">Edit</a>
+  <shipping-option class="shipping-summary__option" [shippingOption]="shippingOption" (updateShippingOption)="onUpdateShippingOption($event)"></shipping-option>
+  <button type="button" class="button large expanded" [disabled]="disableContinueToPayment">Continue to Payment</button>
```

Comment:
Should be computed state off daffodil core. Should determine whether or not the "Shipping Option" is not null.

Reply:
turned out I could just do `[disabled]="!shippingOption"` so I ended up not needing to add a selector